crates/volta-core/src/tool/binary.rs

```diff
@@ -27,29 +29,38 @@ where
                 });
             }
 
+            debug!("Found bin in project at {}", path_to_bin.display());
```

Comment:
Should we say "Found a bin for {} in project at {}" where the first substitution is the name of the bin we are looking for? 

All of the messages in this file _might_ want the same treatment?

Reply:
That's a good idea, I initially omitted it because the `path_to_bin` will have e.g. `/ember` as the end of it, but I think it probably reads better if the command is included separately.

Reply:
Agree, also (not sure if this is true already) we'd want to quote the path and ensure it doesn't get wrapped (wasn't there a util for that already?)

Reply:
There isn't any custom wrapping style applied to the debug output at the moment, so the only wrapping that would happen would be applied by the terminal itself. If / when we add that behavior, we'll definitely use the same logic we already have for the Warning messages that prevents it from wrapping paths on hyphen characters.